ansible/ha-meta.yml

```diff
@@ -0,0 +1,230 @@
+# This playbook gathers meta data facts for SAP/HANA checker.
+
+- hosts: all
```

Comment:
We don't need to run this in `all` hosts.
We have to run it in `localhost`, and remove all of this `delegate_to: localhost` from all the roles. it is redundant

Reply:
done